CLAUDE.md

```
@@ -18,3 +33,23 @@ docker compose -f docker-compose-local.yml up -d  # 인프라 (PostgreSQL 16, Re
+log.warn("[SSE 이벤트 : 전송 실패]: wardKey={} | error={}", wardKey, e.getMessage());
+log.error("[Redis 캐시 : 직렬화 실패]: wardKey={} | error={}", wardKey, e.getMessage());
+```
+
+- 카테고리: 도메인 또는 컴포넌트명 (예: `GPS 수신`, `SSE 이벤트`, `Redis 캐시`)
+- 상세내용: 동작 결과 (예: `저장 완료`, `전송 실패`, `연결 종료`)
+- 추가 정보: `key=value | key=value` 형식으로 컨텍스트 추가
+- 레벨 선택: 정상 흐름 → `info`, 비즈니스 예외 → `warn`, 시스템 오류 → `error`, 디버그 → `debug`
+
+## 인덱스 규칙
+
+Entity `@Table`의 `indexes` 속성을 사용하지 않는다. 인덱스는 별도 DDL 쿼리로 관리하며, 필요한 위치에 TODO 주석으로 표시한다.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)